src/Microsoft.AspNet.Mvc.Razor/Diagnostics/IRuntimeCompilationException.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Framework.Runtime;
+
+namespace Microsoft.AspNet.Diagnostics
+{
+    /// <summary>
+    /// Specifies the contracts for an exception that represents a compilation failure at runtime.
+    /// </summary>
+    [AssemblyNeutral]
+    public interface IRuntimeCompilationException
```

Comment:
Yup. We specifically had a discussion about errors in multiple files at https://github.com/aspnet/Diagnostics/pull/79#discussion_r22322466. If we do want to support that sort of scenario, might make sense to change the interface. Thoughts @davidfowl